javascript/node/README.md

```diff
@@ -0,0 +1,40 @@
+## Node.js Examples
+
+This directory contains several examples of the Splunk Enterprise SDK for JavaScript running
```

Comment:
Can we add guidance to install the required dependencies including the Splunk SDK for Javascript in the `./javascript` directory first by running `npm install`?

Reply:
Actually I would suggest adding a README.md to the `./javascript` directory and populate it with all of the example related sections here: https://github.com/splunk/splunk-sdk-javascript/blob/5d600c1436d8b50954bc07b67db13e24312c92ef/README.md and also add the step above to instruct users to install with `npm install`. How does that sound to you?

Reply:
Yes. It sounds correct to me for adding README inside javascript/ folder. 
But I am thinking of just adding a reference of app examples repo into Examples section of SDK repo README file. We will add reference links of app examples repo if user wants to test examples.

And we have planned changing all these link references as a separate task after migration.